syft/frameworks/torch/hook.py

```diff
@@ -273,19 +275,21 @@ def overloaded_attr(_self, *args, **kwargs):
                 # Store this utility function in the registry
                 hook_self.args_hook_for_overloaded_attr[attr] = hook_args_function
 
-            try:
+            # has_child = hasattr(_self, "child")
+            # if(has_child):
+            #     has_pointer_child = isinstance(_self.child, syft.frameworks.torch.tensors.PointerTensor)
+            # else:
+            #     has_pointer_child = False
+
+            # TODO: change if statement to "if has_pointer_child"
+
+            if not isinstance(_self, syft.frameworks.torch.tensors.PointerTensor):
```

Comment:
Nice trick for methods! Why did you comment the code inspecting the child? Isn't it precisely what we have to do as _self will always be of type torch.Tensor ?

Reply:
Yes - but it broke some other things. In the spirit of "incremental changes" I wanted to merge pieces of the fix earlier.

Reply:
Could you add a comment explaining why this is commented out in the code?